tests/framework/framework.go

```diff
@@ -152,9 +157,8 @@ func (f *Framework) BeforeEach() {
 		createNFSPVs(f.K8sClient, f.CdiInstallNs)
 	}
 
-	defaultFeatureGates := []string{featuregates.HonorWaitForFirstConsumer}
-	ginkgo.By(fmt.Sprintf("Configuring default FeatureGates %q", defaultFeatureGates))
-	f.setFeatureGates(defaultFeatureGates)
+	ginkgo.By(fmt.Sprintf("Configuring default FeatureGates %q", f.FeatureGates))
```

Comment:
Shouldn't this be f.Config.FeatureGates when assigning?

Reply:
`Config` is embedded in `Framework` so `f.FeatureGates` works.  Like `pod.Name` == `pod.ObjectMeta.Name`

Reply:
Right, I missed that. I used that property with the clients to make that work.